paasta_tools/kubernetes_tools.py

```diff
@@ -499,6 +515,13 @@ def raw_selectors_to_requirements(
     return requirements
 
 
+def registration_label(namespace: str) -> str:
+    """ Returns namespace prefixed with regs.{paasta_prefix}/,
+        truncated to 63 characters with hash as suffix if needed.
+    """
+    return limit_size_with_hash(f"regs.{PAASTA_ATTRIBUTE_PREFIX}{namespace}")
```

Comment:
You should replace this with `return(f"regs.{PAASTA_ATTRIBUTE_PREFIX}{limit_size_with_hash(namespace)}")

The 63 character limit is on the suffix part of the label, not the entire length. The entire length can be up to 253 characters in total. Check https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set for that